### DISCUSSION TITLE
Gihuib Copilot Chat Icon is missing from VS Code Left sidebar panel

### DISCUSSION THREAD

**Discussion by fahim9778** [community/community#55731] [Copilot Conversations]
### Select Topic Area

Question

### Body

Recently, in VS code marketplace, I was able to find an extension named "**_Github Copilot Chat_**" from GitHub which was in pre-release. I installed the extension and it was installed without any issues. However, I don't see any chat icon or sidebar in my VS Code left panel. I did it both on VS code normal version and the Insiders version too., but no luck!

Can anyone tell me what could be the possible reason? I remember signing up for the waitlist for Github Copilot Chat somewhere in April or late March.

**Comment by davecheney** [comment]
@fahim9778 the extension is disabled until your waitlist invite arrives. I see you are on the waitlist. When you are invited you will receive an email with instructions for how to get started with Copilot Chat.

**Reply by fahim9778** [reply]
Any ETA? 🥺

**Comment by alindsay55661** [comment]
@davecheney - I am seeing this behavior as well. I have been using Github Copilot and the Copilot Chat extension for some time already. Recently, an organization I was added to enabled Copilot for me and I got an email that said my personal account was canceled automatically. I am still able to use Github Copilot normally (that is, code completions are still happening) but the chat icon in VSCode shows up for a brief moment during "boot" and then disappears. I cannot access the chat window anywhere. What recommendations do you have?

**Reply by alindsay55661** [reply]
I should mention that I have all the latest updates:

- VSCode v1.84.0
- Github Copilot v1.133.0
- Github Copilot Chat v0.10.0

**Reply by alindsay55661** [reply]
I found the problem. Github Copilot Chat is a separate feature that must be turned on in the Github Copilot Organization settings.

![image](https://github.com/community/community/assets/270710/700355dc-1e81-46ad-9496-82cf8debb985)
